zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -70,10 +70,12 @@ use std::ops::Mul;
 
 #[derive(Clone)]
 pub struct Config<F: Field> {
+    pub(crate) selector: Column<Fixed>,
```

Comment:
in rust the idiom way is not to use trait to constrain type in struct definition? 
Only constrain generic type in `impl`?
So remove the ": Field" follows the idiom coding and be more consistent with other parts of the codebase, eg: "EvmCircuit<F>"